.changelog/26156.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:improvement
+cli: Added related evals and placed allocations tables to the eval status command, and exposed more fields without requiring the -verbose flag.
```

Comment:
```suggestion
cli: Added related evals and placed allocations tables to the eval status command, and exposed more fields without requiring the `-verbose` flag.
```

Reply:
Huh, for some reason I didn't think that worked but it totally does and looking at past changelogs I've even done it myself :facepalm: